specifyweb/frontend/js_src/lib/components/FormSliders/IntegratedRecordSelector.tsx

```diff
@@ -6,28 +6,46 @@ import { f } from '../../utils/functools';
 import type { RA } from '../../utils/types';
 import { defined } from '../../utils/types';
 import { DataEntry } from '../Atoms/DataEntry';
+import { fetchCollection } from '../DataModel/collection';
 import {
   DependentCollection,
   LazyCollection,
 } from '../DataModel/collectionApi';
-import type { AnySchema } from '../DataModel/helperTypes';
+import { getField } from '../DataModel/helpers';
+import type { AnySchema, SerializedResource } from '../DataModel/helperTypes';
 import type { SpecifyResource } from '../DataModel/legacyTypes';
 import { resourceOn } from '../DataModel/resource';
+import { schema } from '../DataModel/schema';
 import type { Relationship } from '../DataModel/specifyField';
-import type { Collection } from '../DataModel/specifyModel';
+import type { Collection, SpecifyModel } from '../DataModel/specifyModel';
+import type {
+  Disposal,
+  DisposalPreparation,
+  Gift,
+  GiftPreparation,
+  Loan,
+  LoanPreparation,
+  RecordSet,
+} from '../DataModel/types';
 import { raise } from '../Errors/Crash';
 import { FormTableCollection } from '../FormCells/FormTableCollection';
 import type { FormMode, FormType } from '../FormParse';
 import type { SubViewSortField } from '../FormParse/cells';
 import { augmentMode, ResourceView } from '../Forms/ResourceView';
+import { userInformation } from '../InitialContext/userInformation';
+import { InteractionDialog } from '../Interactions/InteractionDialog';
+import { toSmallSortConfig } from '../Molecules/Sorting';
 import { hasTablePermission } from '../Permissions/helpers';
 import { relationshipIsToMany } from '../WbPlanView/mappingHelpers';
 import type {
   RecordSelectorProps,
   RecordSelectorState,
 } from './RecordSelector';
 import { useRecordSelector } from './RecordSelector';
-
+const defaultOrder: SubViewSortField = {
+  fieldNames: ['timestampCreated'],
+  direction: 'desc',
+};
 // REFACTOR: encapsulate common logic from FormTableCollection and this component
```

Comment:
Remove that refactor statement after you are done
Also, dont' forget to delete the FormTableCollection file